### PR TITLE
OTT-1192: Log wrapper count for vast unwrap request and update existing vast unwrap stat

### PR DESCRIPTION
**Pull request by Pubmatic-Supriya-Patil**
# Description

Please add change description or link to ticket, docs, etc.

# Checklist:

- [ ] PR commit list is unique (rebase/pull with the origin branch to keep master clean).
- [ ] JIRA number is added in the PR title and the commit message.
- [ ] Updated the `header-bidding` repo with appropiate commit id.
- [ ] Documented the new changes.

For Prebid upgrade, refer: https://inside.pubmatic.com:8443/confluence/display/Products/Prebid-server+upgrade

### CODE REVIEW

File: modules/pubmatic/vastunwrap/stats/metrics.go [hunk 1]
```diff
@@ -32,12 +37,16 @@ func NewMetricsEngine(cfg moduledeps.ModuleDeps) *Metrics {
 	metrics.Registry = cfg.Registry
 	metrics.requests = newCounter(cfg, metrics.Registry,
 		"vastunwrap_status",
-		"Count of vast unwrap requests labeled by publisher ID, bidder and status.",
-		[]string{pubIdLabel, bidderLabel, statusLabel})
-	// metrics.requestTime = newHistogramVec(cfg, metrics.Registry,
-	// 	"vastunwrap_request_time",
-	// 	"Time taken to serve the vast unwrap request in Milliseconds", []string{pubIdLabel, bidderLabel},
-	// 	[]float64{50, 100, 200, 300, 500})
+		"Count of vast unwrap requests labeled by bidder and status.",
```
Comment (pm-jaydeep-mohite) [inline modules/pubmatic/vastunwrap/stats/metrics.go:40]: Should it be  `Count of vast unwrap requests labeled by status`  ?

File: modules/pubmatic/vastunwrap/unwrap_service.go [hunk 2]
```diff
@@ -5,24 +5,28 @@ import (
...
 	defer func() {
-		// respTime := time.Since(startTime)
-		// m.MetricsEngine.RecordRequestTime(accountID, bidder, respTime)
 		if r := recover(); r != nil {
 			glog.Error("AdM:" + bid.Bid.AdM + ". stacktrace:" + string(debug.Stack()))
 		}
-		m.MetricsEngine.RecordRequestStatus(accountID, bidder, respStatus)
+		respTime := time.Since(startTime)
+		m.MetricsEngine.RecordRequestTime(bidder, respTime)
+		m.MetricsEngine.RecordWrapperCount(bidder, strconv.Itoa(int(wrapperCnt)))
```
Comment (pm-jaydeep-mohite) [inline modules/pubmatic/vastunwrap/unwrap_service.go:28]: Can we log Wrapper Count only if respStatus = success, so that we would get true inline count ?

Comment (Pubmatic-Supriya-Patil) [reply modules/pubmatic/vastunwrap/unwrap_service.go:28]: yes, I will update code.

File: modules/pubmatic/vastunwrap/stats/metrics.go [hunk 3]
```diff
@@ -32,12 +37,16 @@ func NewMetricsEngine(cfg moduledeps.ModuleDeps) *Metrics {
...
+		[]string{bidderLabel, statusLabel})
+	metrics.wrapperCount = newCounter(cfg, metrics.Registry,
+		"vastunwrap_wrapper_count",
+		"Count of wrapper levels labeled by bidder and wrapper count.",
```
[the remark this answers is not in the document]
Comment (Pubmatic-Supriya-Patil) [reply modules/pubmatic/vastunwrap/stats/metrics.go:44]: you mean Count of vast unwrap levels labeled by bidder?

File: modules/pubmatic/vastunwrap/stats/metrics.go (hunk 1 above)
Comment (Pubmatic-Supriya-Patil) [reply modules/pubmatic/vastunwrap/stats/metrics.go:40]: ok, I will update this

File: modules/pubmatic/vastunwrap/stats/metrics.go (hunk 3 above)
[the remark this answers is not in the document]
Comment (pm-jaydeep-mohite) [reply modules/pubmatic/vastunwrap/stats/metrics.go:44]: yes

**Review by pm-jaydeep-mohite — APPROVED** [review]
LGTM

**Review by pm-viral-vala — APPROVED** [review]

**Pull request merged**